test/unit/specs/lib/error-handler.spec.js

```diff
@@ -0,0 +1,38 @@
+import { errorHandler } from '../../../../src/lib/error-handler'
+
+describe('errorHandler', () => {
+  const errorString = 'errorString'
+  const info = 'additional info provided by vue'
+
+  describe('with Error thrown', () => {
```

Comment:
Can you remove the nested describe blocks? Everywhere else the describe blocks aren't nested

Reply:
👍